contracts/interfaces/IRelayRecipient.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity ^0.5.16;
+
+contract IRelayRecipient {
```

Comment:
I would actually give it a different name, it may be confusing to people what to inherit: IRelayRecipient or BaseRelayRecipient. IForwardable or something in this manner should be better, I think.

Reply:
Actually, they can use IRelayRecipient, without our base-class. I acutally need to bring getSender here too.
(didn't, because it need to be internal, but internals can be in interface just the same)
I don't like the name RelayRecipient - but that's another issue